### PR TITLE
Added openmw

### PR DESCRIPTION
**Pull request by d10sfan**
This adds openmw. For the actual package, it is here: https://gitlab.com/luxtorpeda/packages/openmw. It needs autoconf to work.

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -104,5 +104,11 @@
 	"698780": {
 		"game_name":    "Doki Doki Literature Club",
 		"command":      "./DDLC.sh"
+	},
+	"22320": {
+        "game_name": "The Elder Scrolls III: Morrowind",
```
Comment (dreamer) [inline packages.json]: I know it's hard to spot, but in this specific file I use tabs for indentation, and spaces to line up values in json key: value pairs.

**Comment by d10sfan** [comment]
> Do you want to add OpenMW to README.md file as well, or do you prefer not to advertise the package yet?

I've updated the PR to have the readme updated for openmw, so that once we're ready it can be visible.

**Comment by dreamer** [comment]
OK; in future I will change format of that json file to remove tab-based indentation; Rust coding convention requires spaces so it will be more consistent with other files in the project.

I'll wait with merging the PR until we'll resolve the GitLab build timeouts.

**Comment by dreamer** [comment]
Format of packages.json file changed somewhat since this PR was created. I updated and committed the json part: 2deb5c16a2d99212ae13c92a00a0e89a2ada1f82

Game now starts on my machine, but I am not familiar with OpenMW and can't say if it works as it should (e.g. is there an intro movie?). I think it needs some more work still before we can advertise it as supported in README file (I left some TODOs in the package repo), but if someone wants to test it - then now it'll be possible (just backup your saves if you care about them).

**Pull request closed without merging**